cmd/terramate/cli/cli.go

```diff
@@ -606,14 +606,19 @@ func (c *cli) run() {
 	case "experimental cloud drift show":
 		c.cloudDriftShow()
 	case "script list":
+		c.checkScriptEnabled()
 		c.printScriptList()
 	case "script tree":
+		c.checkScriptEnabled()
 		c.printScriptTree()
-	case "script info <labels>":
+	case "script info <cmds>":
```

Comment:
Since you are modifying this part and improving the errors, could add an additional `case "script info"` without args here, analogous this code below:
```
case "script run":
     c.checkScriptEnabled()
     log.Fatal().Msg("no script specified")
```

Reply:
sure

Reply:
fixed it, but I had to make the args `optional:"true"`. Just noticed that `terramate run` without args is not falling in the "run" case, for similar reason. Will be fixed separately.